docs/sources/enterprise/enterprise-configuration.md

```diff
@@ -69,19 +69,29 @@ Set to complete URL to override Apple/iOS icon.
 
 List the link IDs to use here. Grafana will look for matching link configurations, the link IDs should be space-separated and contain no whitespace.
 
-## [meta_analytics]
+## [usage_insights.export]
 
```

Comment:
Add an introductory/stem sentence. The style guide suggests we should not stack the headings.